jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/JibIntegrationTest.java

```diff
@@ -109,6 +114,64 @@ public void testScratch()
         inspectOutput.contains("\"Layers\": ["));
   }
 
+  @Test
+  public void testOffline()
+      throws IOException, InterruptedException, InvalidImageReferenceException, ExecutionException,
+          RegistryException, CacheDirectoryCreationException {
+    LocalRegistry tempRegistry = new LocalRegistry(5001);
```

Comment:
`LocalRegistry` is already defined as a `@Rule` in this test class. Can't we use that? Looks like we can. It solves another minor problem that the registry will stop even if the test fails. (It is better to do `tempRegistry.stop()` in `finally` if you don't want to reuse it.)

Reply:
The reason is because I want to manually stop it in the middle of the test to make sure everything still works, even if the registry can't be reached.

Reply:
Ah, okay.